### ISSUE TITLE
Add {switch} example in Fluid Field's Grid Example

### ISSUE THREAD

**Issue by bsrweb** [ExpressionEngine/ExpressionEngine-User-Guide#9]
**Suggested Change**
Add a single-line example of a {switch} statement inside a Fluid Field's Grid Field code example.

**Reasoning**
The syntax is a little different when a Grid Field is inside a Fluid Field loop:
`{content:switch='odd|event]` as opposed to a stand-along Grid Field: `{grid_field_name:switch='odd|even'}`.

**Additional context**
Or maybe it's just me. When I first ran into this, I tried `{content}{grid_field:switch}{/content}` , then `{content:grid_field:switch}`, and then after a process of elimination, got the `{switch}` to work.

**Comment by derekjones** [comment]
Good idea, @bsrweb! Want to open a pull request in the [User Guide Repo](https://github.com/ExpressionEngine/ExpressionEngine-User-Guide) with your suggested changes?

**Comment by derekjones** [comment]
> Yes, I'd be happy to contribute. I just need to figure out how Docker works first.

We've abstracted a lot of that way, via the `eetools` command line tool, so if you install Docker Desktop and follow the readme instructions in this repo, you should be good to go! If you're on Windows 10, I _think_ it's Bash shell should still work with it? But I've not personally tested it.
